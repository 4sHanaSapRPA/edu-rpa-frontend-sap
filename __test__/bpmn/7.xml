<?xml version="1.0" encoding="UTF-8"?>
<bpmn:definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_0jl7jys" targetNamespace="http://bpmn.io/schema/bpmn" exporter="bpmn-js (https://demo.bpmn.io)" exporterVersion="14.0.0">
  <bpmn:process id="Process_04og7m0" isExecutable="false">
    <bpmn:startEvent id="StartEvent_1mxxylx">
      <bpmn:outgoing>Flow_12ctyks</bpmn:outgoing>
    </bpmn:startEvent>
    <bpmn:sequenceFlow id="Flow_12ctyks" sourceRef="StartEvent_1mxxylx" targetRef="Gateway_0uqg7tr" />
    <bpmn:task id="Activity_1536xo5" name="Get Title Assign to ${title}">
      <bpmn:incoming>Flow_19fnu56</bpmn:incoming>
      <bpmn:incoming>Flow_0a8x10v</bpmn:incoming>
      <bpmn:outgoing>Flow_0vxkb6m</bpmn:outgoing>
    </bpmn:task>
    <bpmn:task id="Activity_012ckfs" name="Close Browser">
      <bpmn:incoming>Flow_0vxkb6m</bpmn:incoming>
      <bpmn:outgoing>Flow_0piu48d</bpmn:outgoing>
    </bpmn:task>
    <bpmn:endEvent id="Event_10jy91v">
      <bpmn:incoming>Flow_0piu48d</bpmn:incoming>
    </bpmn:endEvent>
    <bpmn:sequenceFlow id="Flow_0piu48d" sourceRef="Activity_012ckfs" targetRef="Event_10jy91v" />
    <bpmn:exclusiveGateway id="Gateway_0uqg7tr">
      <bpmn:incoming>Flow_12ctyks</bpmn:incoming>
      <bpmn:outgoing>Flow_0haw5vn</bpmn:outgoing>
      <bpmn:outgoing>Flow_19fnu56</bpmn:outgoing>
    </bpmn:exclusiveGateway>
    <bpmn:sequenceFlow id="Flow_0haw5vn" sourceRef="Gateway_0uqg7tr" targetRef="Activity_0pkjxtm" />
    <bpmn:sequenceFlow id="Flow_19fnu56" sourceRef="Gateway_0uqg7tr" targetRef="Activity_1536xo5" />
    <bpmn:sequenceFlow id="Flow_0vxkb6m" sourceRef="Activity_1536xo5" targetRef="Activity_012ckfs" />
    <bpmn:task id="Activity_0pkjxtm" name="Open Browser">
      <bpmn:incoming>Flow_0haw5vn</bpmn:incoming>
      <bpmn:outgoing>Flow_0a8x10v</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_0a8x10v" sourceRef="Activity_0pkjxtm" targetRef="Activity_1536xo5" />
  </bpmn:process>
  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_04og7m0">
      <bpmndi:BPMNShape id="Activity_1536xo5_di" bpmnElement="Activity_1536xo5">
        <dc:Bounds x="560" y="80" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Activity_0pkjxtm_di" bpmnElement="Activity_0pkjxtm">
        <dc:Bounds x="370" y="80" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="BPMNShape_1vqtd7o" bpmnElement="Gateway_0uqg7tr" isMarkerVisible="true">
        <dc:Bounds x="255" y="95" width="50" height="50" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="StartEvent_1mxxylx">
        <dc:Bounds x="142" y="102" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="BPMNShape_0eltz4d" bpmnElement="Activity_012ckfs">
        <dc:Bounds x="810" y="80" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Event_10jy91v_di" bpmnElement="Event_10jy91v">
        <dc:Bounds x="1072" y="102" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNEdge id="Flow_12ctyks_di" bpmnElement="Flow_12ctyks">
        <di:waypoint x="178" y="120" />
        <di:waypoint x="255" y="120" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0piu48d_di" bpmnElement="Flow_0piu48d">
        <di:waypoint x="910" y="120" />
        <di:waypoint x="1072" y="120" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0haw5vn_di" bpmnElement="Flow_0haw5vn">
        <di:waypoint x="305" y="120" />
        <di:waypoint x="370" y="120" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0a8x10v_di" bpmnElement="Flow_0a8x10v">
        <di:waypoint x="470" y="120" />
        <di:waypoint x="560" y="120" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_19fnu56_di" bpmnElement="Flow_19fnu56">
        <di:waypoint x="280" y="145" />
        <di:waypoint x="280" y="280" />
        <di:waypoint x="610" y="280" />
        <di:waypoint x="610" y="160" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0vxkb6m_di" bpmnElement="Flow_0vxkb6m">
        <di:waypoint x="660" y="120" />
        <di:waypoint x="810" y="120" />
      </bpmndi:BPMNEdge>
    </bpmndi:BPMNPlane>
  </bpmndi:BPMNDiagram>
</bpmn:definitions>
