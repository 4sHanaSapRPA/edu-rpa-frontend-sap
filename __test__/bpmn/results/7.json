{
  "block": [
    {
      "start": "Gateway_0uqg7tr",
      "braches": [
        {
          "block": []
        },
        {
          "block": [
            {
              "id": "Activity_0pkjxtm",
              "name": "Open Browser",
              "incomming": [
                "Flow_0haw5vn"
              ],
              "outgoing": [
                "Flow_0a8x10v"
              ]
            }
          ]
        }
      ]
    },
    {
      "id": "Activity_1536xo5",
      "name": "Get Title Assign to ${title}",
      "incomming": [
        "Flow_19fnu56",
        "Flow_0a8x10v"
      ],
      "outgoing": [
        "Flow_0vxkb6m"
      ]
    },
    {
      "id": "Activity_012ckfs",
      "name": "Close Browser",
      "incomming": [
        "Flow_0vxkb6m"
      ],
      "outgoing": [
        "Flow_0piu48d"
      ]
    }
  ]
}