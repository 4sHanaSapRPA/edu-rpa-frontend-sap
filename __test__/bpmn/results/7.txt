
IF: Gateway_0uqg7tr
	Activity_1536xo5 - Get Title Assign to ${title}
	Activity_012ckfs - Close Browser
ELSE:
	Activity_0pkjxtm - Open Browser
	Activity_1536xo5 - Get Title Assign to ${title}
	Activity_012ckfs - Close Browser
