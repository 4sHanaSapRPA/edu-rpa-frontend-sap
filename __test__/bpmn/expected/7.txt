IF: Gateway_0uqg7tr
	[Empty]
ELSE:
	Activity_0pkjxtm - Open Browser

Activity_1536xo5 - Get Title Assign to ${title}
Activity_012ckfs - Close Browser